lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -62,6 +62,8 @@ defmodule ExUnit.Assertions do
   or handling exceptions.
   """
 
+  alias ExUnit.Pattern
+
   @doc """
```

Comment:
There are a few examples in the documentation that should include the new `left:` key.

Reply:
Hi, would you elaborate please, I don't understand.

Reply:
Sure.

For instance, [this example](
https://github.com/elixir-lang/elixir/pull/8002/files#diff-699b0020c6a196231a7d473322f1744dR91) should be updated to:

```elixir
match (=) failed
code:  [1] = [2]
left:  [1]
right: [2]
```

like you did in [this test](
https://github.com/elixir-lang/elixir/pull/8002/files#diff-dcedcba19e3df61c281c3e36e50e53d9R797).